src/components/v1/shared/BudgetForm.tsx

```diff
@@ -51,7 +52,7 @@ export default function BudgetForm({
   const terminalFee = useTerminalFee(terminal?.version)
 
   useLayoutEffect(() => {
-    setCurrency(initialCurrency)
+    setCurrency(V1CurrencyName(initialCurrency) ?? 'ETH')
```

Comment:
lets use the constants for these strings where possible. it means if we ever need to change them, we only need to change one spot instead of many.

Reply:
Don't think we have constants for these yet. So got a housekeeping issue in #808